qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddUserTest.java

```diff
@@ -38,26 +34,9 @@ public void addUserTest() {
             manageUserPage.removeUser("super_nurse");
         } else {
             AddEditUserPage addEditUserPage = manageUserPage.clickOnAddUser();
-            addEditUserPage.createNewPerson();
-
-            addEditUserPage.saveUser();
-            List<String> validationErrors = addEditUserPage.getValidationErrors();
-            assertTrue(validationErrors.contains("You must define at least one name"));
-            assertTrue(validationErrors.contains("Cannot be empty or null"));
-            assertFalse(addEditUserPage.isDataCorrect(validationErrors));
-
-            addEditUserPage.enterGivenFamily("Super", "Nurse");
-            addEditUserPage.saveUser();
```

Comment:
Are you sure this is working fine

Reply:
> Are you sure this is working fine

@sherrif10 it is working fine on my side, I have just rerun it again. I just want to look into the reason why it's failing here.